### PR TITLE
Fix Season folders

### PR DESCRIPTION
**Pull request by fernandog**
@miigotu like this?

### CODE REVIEW

**Comment by SiCKRAGEB0T** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Give a description on what the PR is for.
- [x] Make sure you send a PR against the DEVELOP branch
- [x] Don't send big changes in one go. Split up big PRs into multiple smaller (easier manageble) PRs

Please make sure you also read [contribution guide](https://github.com/SiCKRAGETV/SickRage/blob/master/contributing.md) and followed all the steps.

Thanks!

**Comment by fernandog** [comment]
@miigot it's no working, editing show is queuing a refresh to the show

File: gui/slick/views/displayShow.mako [hunk 1]
```diff
@@ -214,7 +214,7 @@
                     % if sickbeard.USE_SUBTITLES:
                     <tr><td class="showLegend">Subtitles: </td><td><img src="${srRoot}/images/${("no16.png", "yes16.png")[bool(show.subtitles)]}" alt="${("N", "Y")[bool(show.subtitles)]}" width="16" height="16" /></td></tr>
                     % endif
-                    <tr><td class="showLegend">Flat Folders: </td><td><img src="${srRoot}/images/${("no16.png", "yes16.png")[bool(show.flatten_folders or sickbeard.NAMING_FORCE_FOLDERS)]}" alt=="${("N", "Y")[bool(show.flatten_folders or sickbeard.NAMING_FORCE_FOLDERS)]}" width="16" height="16" /></td></tr>
+                    <tr><td class="showLegend">Season Folders: </td><td><img src="${srRoot}/images/${("no16.png", "yes16.png")[bool(not show.flatten_folders or sickbeard.NAMING_FORCE_FOLDERS)]}" alt=="${("N", "Y")[bool(show.flatten_folders or sickbeard.NAMING_FORCE_FOLDERS)]}" width="16" height="16" /></td></tr>
```
Comment (miigotu) [inline gui/slick/views/displayShow.mako]: At least this one is inconsistant. Not sure what sickbeard.NAMING_FORCE_FOLDERS is either

File: gui/slick/views/manage_massEdit.mako [hunk 2]
```diff
@@ -95,12 +95,12 @@
...
 <div class="optionWrapper clearfix">
-<span class="selectTitle">Flatten Folders <span class="separator">*</span></span>
+<span class="selectTitle">Season folders<span class="separator">*</span></span>
     <div class="selectChoices">
         <select id="edit_flatten_folders" name="flatten_folders" class="form-control form-control-inline input-sm">
             <option value="keep" ${('', 'selected="selected"')[flatten_folders_value == None]}>&lt; keep &gt;</option>
-            <option value="enable" ${('', 'selected="selected"')[flatten_folders_value == 1]}>enable</option>
-            <option value="disable" ${('', 'selected="selected"')[flatten_folders_value == 0]}>disable</option>
+            <option value="enable" ${('', 'selected="selected"')[flatten_folders_value == 1]}>disable</option>
+            <option value="disable" ${('', 'selected="selected"')[flatten_folders_value == 0]}>enable</option>
```
Comment (miigotu) [inline gui/slick/views/manage_massEdit.mako]: These feel wrong too. Maybe we need to take the change deeper.

**Comment by fernandog** [comment]
I think now it's ok. 
@SiCKRAGETV/moderators can someone else test this please?

**Pull request merged**